src/Symfony/Component/Notifier/Exception/MultipleExclusiveOptionsUsedException.php

```diff
@@ -16,15 +16,11 @@
  */
 class MultipleExclusiveOptionsUsedException extends InvalidArgumentException
 {
-    /**
-     * @param string[]      $usedExclusiveOptions
-     * @param string[]|null $exclusiveOptions
-     */
-    public function __construct(array $usedExclusiveOptions, array $exclusiveOptions = null, \Throwable $previous = null)
+    public function __construct(array $usedOptions, array $availableOptions = null, \Throwable $previous = null)
     {
-        $message = sprintf('Multiple exclusive options have been used "%s".', implode('", "', $usedExclusiveOptions));
-        if (null !== $exclusiveOptions) {
-            $message .= sprintf(' Only one of %s can be used.', implode('", "', $exclusiveOptions));
+        $message = sprintf('Multiple exclusive options have been used "%s".', implode('", "', $usedOptions));
+        if (null !== $availableOptions) {
```

Comment:
```suggestion
        if (count($exclusiveOptions) > 0) {
```
If we will make second argument not nullable, we should check that array not empty here, or remove this check at all and generate full message on previous line.

Reply:
I removed the check